scripts/ci/lint

```diff
@@ -52,6 +52,19 @@ echo "::group::Checking eol"
 checkerr "$(check-eol-dry-run)"
 echo "::endgroup::"
 
+check-spdx()
+{
+  for file in $(git ls-files -- ":/" ":/!:libopencm3" ":/!:LICENSE" ":/!:fips202/LICENSE" ":/!:mlkem/LICENSE" ":/!:hal/mps2/LICENSE.txt" ":/!:.git*" ":/!:flake.lock"); do
```

Comment:
maybe you would like to change 
```
":/!:LICENSE" ":/!:fips202/LICENSE" ":/!:mlkem/LICENSE" ":/!:hal/mps2/LICENSE.txt"
```
to something like
```
":/!:*LICENSE*"
```

Reply:
Thanks. Good idea.